package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@centrapay/jest-date-matchers",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
@seliwe No lockfile change?

Reply:
you mean yarn.lock? It only contains dependency versions, didn't change after a `yarn install`